sub/sd_lavc.c

```diff
@@ -195,9 +195,12 @@ static void read_sub_bitmaps(struct sd *sd, struct sub *sub)
             MP_ERR(sd, "unsupported subtitle type from libavcodec\n");
             continue;
         }
-        if (!(r->flags & AV_SUBTITLE_FLAG_FORCED) && (opts->forced_subs_only == 1 ||
-            (opts->forced_subs_only && sd->forced_only_def)))
+        if (r->flags & AV_SUBTITLE_FLAG_FORCED) {
+            sd->has_forced_subevents = true;
```

Comment:
is the entire subtitle data read ahead of time? (I don't think so)
if not, how does updating this at runtime work? at least osc.lua would need to be able to observe it

Reply:
I misunderstood what this for loop did, and mistakenly thought it read reading the whole thing ahead of time. And the reason why this worked is because the sample both me and @Dudemanguy were testing with has the forced track tagged as forced.

I'll try to figure out some other way of detecting this, but if not I think it's better to just remove this osc button entirely

Reply:
> the sample both me and @Dudemanguy were testing with has the first subpicture tagged as forced.

can you link it somewhere next time?